pkg/web/templates/components/nav.html

```diff
@@ -27,9 +27,10 @@
    <div class="dropdown dropdown-bottom dropdown-left">
     <div tabindex="0" role="button" class="mr-4">
       <i class="fa-solid fa-user text-lg"></i>
-      <span class="sr-only">Open User Settings Menu</span>
+      <span class="sr-only">Open Settings Menu</span>
```

Comment:
Should we add screen reader instructions for the other menu items? Is that what `sr-only` means? What is the role of the icon above?

Reply:
Yes that's what `sr-only` stands for. This was added because we only have an icon on the page, so adding this would help explain what the icon does. 

The menu items have text next to the icons that explain each menu option, so that's why `sr-only` wasn't used there.